### ISSUE TITLE
Widget forgets configuration after reboot 

### ISSUE THREAD

**Issue by FedeDP** [Misenko/rssnow-plasma5-applet#2]
Hi!
I've got a bug for you too :)
It seems the widget resets its configuration every boot; plus on second boot it cannot connect. I have to remove it and add a new widget:

![screenshot_20151029_152001](https://cloud.githubusercontent.com/assets/5837210/10821003/6242e75e-7e50-11e5-80b6-8d20fc7272bb.png)

**Comment by amarthadan** [comment]
Thanks! I'll look into that as well :)

**Comment by FedeDP** [comment]
Well, in past days these problems seem gone, or well...it just takes really long to update the feeds first time it boots up(something like an hour or so, i'd guess it takes "Update interval" minutes) .
At least configuration is saved across reboots.

**Comment by amarthadan** [comment]
You are right. I didn't really find the described configuration bug but I did find the bug in the feed loading mechanism. I just didn't have time to fix it. I'll do it as soon as possible.
